_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+
+## Team constellation and focus
+
+Just recently, our team was restructured. New team members were added, whereas some colleagues moved to other teams. Doing a team offsite was just the right setting to get to know each other better, and boost the team spirit. As we are facing new challenges, one big topic was focusing on our new tasks.
+
+## Workshops
+
+{% image blog/blog-team-ep6.jpg 50% right %}
+
+In order to spread knowledge and improve our [T-shaped skills](https://en.wikipedia.org/wiki/T-shaped_skills) we prepared various workshops.
+In advance we identified topics that are relevant to everyone in the team and topics that are only relevant to a certain guild.
+To make best use of the limited time we held some workshops on parallel tracks.
+
+Of interest to everyone was the "JavaScript in epages 6" session chaired by Markus. He explained in detail how different JavaScript frameworks are integrated in epages 6 and what to consider when writing JavaScript in this environment. Using live coding he demonstrated how the template engine, localization and JavaScript compression work.
+
+{% image blog/blog-ep6-frontend-workshop.jpg 50% left %}
+
+Oliver explained selected [OWASP threats](https://www.owasp.org/index.php/Cheat_Sheets). Afterwards we tried to exploit our software. While we identified some problems, we also concluded that epages 6 is pretty resilient against the most common attacks: injections and cross-site scripting.
+
+Peter outlined our current development pipeline process used primarily for QA and collaboration on more complex features.
+Additionally he clarified the different build  and configuration setups on our team VMs.
+
+On the second day we started with the parallel sessions: UI development and tax calculation in epages 6.
+For the frontend developers and designers Björn presented the various HTML template sections of a typical epages 6 webpage. In addition to that he documented the directory and file structure used for styling as well as the style guide itself.
+The backend folks attended the tax calculation lecture by Michael. He explained in detail how to alter tax settings in epages 6 and which pitfalls to consider when customers and merchants from different countries do business together.
+
+Last but not least Tobias emphasized unit testing and best practices like the [Test Pyramid](http://martinfowler.com/bliki/TestPyramid.html). Using multiple code examples he demonstrated how mocking can be used for epages 6 integration tests.
+
+## Bow and Arrow Action
+
+{% image blog/blog-ep6offsite-bow.jpg 50% right %}
+
+After one and a half day of intense workshops in the Creative Room of the Volkshochschule Jena, we had our recreational team event on Wednesday afternoon. The [archery site](http://camburger-bogenschuetzen.de/) was located in the forest area in Camburg.
```

Comment:
After one and a half day of intense workshops in the Creative Room of the Volkshochschule Jena... > After one and a half day of intense workshops, ...